tests/migrator/test_journals.py

```diff
@@ -290,42 +295,130 @@ def test_migration(app):
         )
         check_transformation(
             """
-            <datafield tag="780" ind1=" " ind2=" ">
-                <subfield code="i">Continues</subfield>
-                <subfield code="t">Mathematics of the USSR-Izvestiya</subfield>
-                <subfield code="w">2711756</subfield>
-            </datafield>
-            """,
+                <datafield tag="787" ind1=" " ind2=" ">
+                    <subfield code="i">Random text</subfield>
+                    <subfield code="w">7483924</subfield>
+                </datafield>
+                """,
+            {
+                "_migration": {
+                    "is_multipart": False,
+                    "record_type": "journal",
+                    "volumes": [],
+                    "items": [],
+                    "electronic_items": [],
+                    "has_related": True,
+                    "related": [
+                        {
+                            "related_recid": "7483924",
+                            "relation_type": "language",
+                        }
+                    ],
+                    "relation_next": None,
+                    "relation_previous": None,
+                },
+            },
+        )
+        check_transformation(
+            """
+                <datafield tag="770" ind1=" " ind2=" ">
+                    <subfield code="i">Random text</subfield>
+                    <subfield code="w">7483924</subfield>
+                </datafield>
+                """,
+            {
+                "_migration": {
+                    "is_multipart": False,
+                    "record_type": "journal",
+                    "volumes": [],
+                    "items": [],
+                    "electronic_items": [],
+                    "has_related": True,
+                    "related": [
+                        {
+                            "related_recid": "7483924",
+                            "relation_type": "other",
```

Comment:
as in the comment above, for the relation other we are missing the description:
subfield `i` -> `relation_note`